runtime/Cargo.toml

```diff
@@ -74,17 +74,20 @@ deno_websocket.workspace = true
 deno_webstorage.workspace = true
 node_resolver = { workspace = true, features = ["sync"] }
 
+async-trait.workspace = true
```

Comment:
Do we actually need async trait crate still?

Reply:
Yes because `BundleProvider` has an async method, but you want it to be dyn-compatible. Without `async-trait` a trait with an async method can't be used with `dyn`.